server/routes/appointments/video-link-booking/handlers/extraInformation.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response } from 'express'
+import { Expose } from 'class-transformer'
+import { IsOptional, MaxLength } from 'class-validator'
+import BookAVideoLinkService from '../../../../services/bookAVideoLinkService'
+
+export class ExtraInformation {
+  @Expose()
+  @MaxLength(4000, { message: 'You must enter extra information which has no more than 4,000 characters' })
```

Comment:
Can you restrict this to 3600?

This is what old BVLS and whereabouts has (ideally we want to reduce the size of this even further).